db/seeds.rb

```diff
@@ -52,7 +54,8 @@
   reason_to_donate: 'I feel like diversity in tech is important',
   slug: 'black-girls-code',
   info_url: 'http://www.blackgirlscode.com',
-  donation_url: 'http://www.blackgirlscode.com'
+  donation_url: 'http://www.blackgirlscode.com',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.